examples/flutter_gallery/test/smoke_test.dart

```diff
@@ -128,6 +128,9 @@ Future<Null> runSmokeTest(WidgetTester tester) async {
     final Finder finder = findGalleryItemByRouteName(tester, routeName);
     Scrollable.ensureVisible(tester.element(finder), alignment: 0.5);
     await tester.pumpAndSettle();
+    // The backdrop demo has no back button so it can't be smoked.
+    if (routeName == '/material/backdrop')
```

Comment:
How do you exit this demo on iOS? 

I hit the same problem on #15324 and after chatting with @yjbanov, we'll add a tester.backPage() or something. But it might not help here.

Reply:
A user could exit on iOS with an edge swipe.

If we had tester.backPage() to use as a fallback when no back button was found, then this demo could be smoked.

Reply:
Seems a bit strange that there'd be a hamburger menu or a close button on the corner and a swipe triggers neither and closes everything. Should there be an explicit exit demo button somewhere instead? (which then can be Tooltipped and smoke tested)

Reply:
I added a back button to the backdrop. It seems a little out of place, but it does give the iOS user a relatively obvious escape route. And it smokes.